react-loops.js

```diff
@@ -60,6 +60,16 @@ var iterall = require("iterall");
  *     }/>
  *   </ul>
  *
+ * * fallback in Loops*
+ * use the prop `fallback` to provide an fallback component, when we got a empty list.
+ * 
+ *   <ul>
+ *     <For of={myList}  fallback={_ => <div>fallback UI</div>}>
```

Comment:
let's bike-shed on the name of this new prop. I think `fallback` makes sense, but are there alternatives we should consider? What do you think about `whenEmpty` or `ifEmpty`

Reply:
I will reconsider this